pom.xml

```
@@ -14,8 +14,8 @@
     <properties>
         <changelist>999999-SNAPSHOT</changelist>
         <!-- https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/ -->
-        <jenkins.baseline>2.346</jenkins.baseline>
-        <jenkins.version>${jenkins.baseline}.3</jenkins.version>
+        <jenkins.baseline>2.479</jenkins.baseline>
+        <jenkins.version>${jenkins.baseline}.1</jenkins.version>
```

Since [choosing a Jenkins version](https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/) recommends 2.479.3, shouldn't this be ".3"?

We've released 2.492.1 and are preparing for the release of 2.492.2.  It seems safe enough to require the most recent release of the 2.479.x line.
```suggestion
        <jenkins.version>${jenkins.baseline}.3</jenkins.version>
```